### PR TITLE
Change log for v1.0.0rc1

### PR DESCRIPTION
**Pull request by ttmc**
Did I miss anything important?
Are there any errors?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,14 +15,71 @@ For reference, the possible headings are:
 * **External Contributors** to list contributors outside of BigchainDB GmbH.
 * **Notes**
 
+## [1.0.0rc1] - 2017-06-??
+Tag name: v1.0.0rc1
+
+### Added
+* Support for SSL/TLS auth and communications between MongoDB and BigchainDB, MongoDB Backup Agent, MongoDB Monitoring Agent, and MongoDB Cloud Manager. Pull Requests
```
Comment (krish7919) [inline CHANGELOG.md]: This should be `Support for secure TLS/SSL communication between MongoDB and BigchainDB, MongoDB Backup Agent, MongoDB Monitoring Agent`.
1. There is no TLS between MongoDB instance and Cloud Manager directly.
2. There is no `auth` support till we merge #1552. We are blocked on Cloud Manager backups using auth.

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,10 +89,10 @@ Tag name: v0.10.2
 ## [0.10.1] - 2017-04-19
 Tag name: v0.10.1
 
-## Added
+### Added
 * Documentation for the BigchainDB settings `wsserver.host` and `wsserver.port`. [Pull Request #1408](https://github.com/bigchaindb/bigchaindb/pull/1408)
 
-## Fixed
+### Fixed
 * Fixed `Dockerfile`, which was failing to build. It now starts `FROM python:3.6` (instead of `FROM ubuntu:xenial`). [Pull Request #1410](https://github.com/bigchaindb/bigchaindb/pull/1410)
```
Comment (krish7919) [inline CHANGELOG.md]: I would reword this as:
Fixed `Dockerfile`, which was failing to build. 
It is based on the `python:3.6` image, rather than the `ubuntu:xenial` image.

Comment (ttmc) [inline CHANGELOG.md]: This line is from the change log for version 0.10.1 and I think it was accurate then.

Would you like to say anything about the (many) `Dockerfile` changes that happened between 0.10.2 and 1.0.0rc1? (The change log is supposed to be about big changes or breaking changes.)
